addons/website_event_booth_sale/static/src/js/booth_register.js

```diff
@@ -0,0 +1,27 @@
+odoo.define('website_event_booth_sale.booth_registration', function (require) {
+'use strict';
+
+const BoothRegistration = require('website_event_booth.booth_registration');
+
+BoothRegistration.include({
```

Comment:
Not sure, what are we doing in this file ? Could probably gain some comments as it is not crystal clear when reading it :) .